bot/exts/filtering/_filter_lists/extension.py

```diff
@@ -20,7 +28,7 @@
     f"please use a code-pasting service such as {PASTE_URL}"
 )
 
-TXT_LIKE_FILES = {".txt", ".csv", ".json"}
+TXT_LIKE_FILES = {".txt", ".csv", ".json", ".py"}
```

Comment:
If we're lifting the ban on `.py`, then anything adjacent feels like it might as well also be added. Maybe `.js`,  `.css`, `.html` so that anyone using Python to build a webapp of some sort won't run into a brick wall? What about stuff like `.yaml`, `.toml`?

Reply:
I realized later that that code has no effect on which file types are allowed.